### ISSUE TITLE
jsonloader: crashes if packageVerificationCodeExcludedFiles is not present

### ISSUE THREAD

**Issue by swinslow** [spdx/tools-golang#80]
cc @specter25 

In the `json` branch, jsonloader is crashing when trying to parse a package that contains no `packageVerificationCodeExcludedFiles` property in the `packageVerificationCode` object.

Sample file: [example1-modded.txt](https://github.com/spdx/tools-golang/files/6682687/example1-modded.txt)  (note that this is a JSON file, but GitHub won't let me upload it with the `.json` extension)

Output when running the `8-jsonloader` example on this file:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x98 pc=0x4df830]

goroutine 1 [running]:
github.com/spdx/tools-golang/jsonloader2v2.JSONSpdxDocument.parseJsonPackages2_2(0xc0000a0150, 0x512216, 0x8, 0x4f00c0, 0xc0000ae120, 0xc0000ca000, 0x0, 0x0)
	/home/steve/programming/spdx/tools-golang/jsonloader2v2/parse_package.go:141 +0x2450
github.com/spdx/tools-golang/jsonloader2v2.JSONSpdxDocument.newDocument(0xc0000a0150, 0xc0000ca000, 0xda2, 0x503b80)
	/home/steve/programming/spdx/tools-golang/jsonloader2v2/jsonloader.go:88 +0x73f
github.com/spdx/tools-golang/jsonloader2v2.(*spdxDocument2_2).UnmarshalJSON(0xc0000ca000, 0xc0000c2000, 0xda1, 0xda2, 0x7f0c4dd39f70, 0xc0000ca000)
	/home/steve/programming/spdx/tools-golang/jsonloader2v2/jsonloader.go:40 +0x9d
encoding/json.(*decodeState).object(0xc0000cc000, 0x4f5a60, 0xc0000ca000, 0x16, 0xc0000cc028, 0x7b)
	/snap/go/7736/src/encoding/json/decode.go:609 +0x20a3
encoding/json.(*decodeState).value(0xc0000cc000, 0x4f5a60, 0xc0000ca000, 0x16, 0xc00009ad70, 0x4d1d4b)
	/snap/go/7736/src/encoding/json/decode.go:370 +0x6d
encoding/json.(*decodeState).unmarshal(0xc0000cc000, 0x4f5a60, 0xc0000ca000, 0xc0000cc028, 0x0)
	/snap/go/7736/src/encoding/json/decode.go:180 +0x1f4
encoding/json.Unmarshal(0xc0000c2000, 0xda1, 0xda2, 0x4f5a60, 0xc0000ca000, 0xda1, 0xda2)
	/snap/go/7736/src/encoding/json/decode.go:107 +0x11a
github.com/spdx/tools-golang/jsonloader2v2.Load2_2(0xc0000c2000, 0xda1, 0xda2, 0xda1, 0xda2, 0x0)
	/home/steve/programming/spdx/tools-golang/jsonloader2v2/jsonloader.go:22 +0xad
main.main()
	/home/steve/programming/spdx/tools-golang/examples/8-jsonloader/examplejsontotv.go:38 +0x23f
```

It looks like the top of the stack is at `parse_package.go` line 141:

https://github.com/spdx/tools-golang/blob/ca9e0d9572b07cc54a1690dd95f2aac901d2439c/jsonloader2v2/parse_package.go#L139-L148

Note that package verification codes might not have an excluded files property, so you'll want to check whether it is present before testing or using it.

**Comment by swinslow** [comment]
@swinslow to check and confirm this is fixed following #83

**Comment by swinslow** [comment]
Confirmed fixed in #83
